app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/AMLSController.scala

```diff
@@ -63,7 +82,7 @@ class AMLSController @Inject()(
         amlsForm(amlsBodies.keys.toSet)
           .bindFromRequest()
           .fold(
-            formWithErrors => Future.successful(Ok(html.money_laundering_compliance(formWithErrors, amlsBodies))),
+            formWithErrors => toFuture(Ok(html.money_laundering_compliance(formWithErrors, amlsBodies))),
             validForm => {
               val amlsDetails = AMLSDetails(
                 amlsBodies.getOrElse(validForm.amlsCode, throw new Exception("Invalid AMLS code")),
```

Comment:
On submission, this looks like it redirects directly to the `showCheckAnswers` page without being subjected to the check for clean/unclean credentials (https://confluence.tools.tax.service.gov.uk/display/AG/Subscription+-+journey+and+pages). Could we use a similar routing to the one that the /business-email submission endpoint uses?